### PR TITLE
NMT Merge

### PR DESCRIPTION
**Pull request by mtgnoah**
This is the work of @bianyuanop and he worked on implementing NMT as well as Ash support for SEQ.

### CODE REVIEW

File: .token-cli/000372.sst
Comment (bianyuanop) [inline .token-cli/000372.sst:1]: I think we should remove `.token-cli` and put this to .gitignore

File: build/token-feed
Comment (bianyuanop) [inline build/token-feed:1]: and the `build` folder should be removed too

**Review by bianyuanop — APPROVED** [review]
All good, and i've run the e2e tests locally, all passed

**Comment by mtgnoah** [comment]
> All good, and i've run the e2e tests locally, all passed

Any ideas on how goreleaser can be fixed?

**Pull request merged**